app/logic/fileHandler.py

```diff
@@ -15,11 +15,17 @@ def __init__(self,files=None, courseId=None, eventId=None):
             # eventID is not included in the path, because it is now a part of the attachment filename.
             self.path = os.path.join(self.path, app.config['files']['event_attachment_path'])
         
+    def makeDirectory(self):
+        # Creating the directory when call in saveFiles
         try:
             extraDir = str(self.eventId) if self.eventId else ""
             os.makedirs(os.path.join(self.path, extraDir))
-        except:
-            print("Directory exists.")
+        # Passing if the error is 17
```

Comment:
This comment isn't useful. I can tell that you are looking at error 17 from the code. But I can't tell what error 17 is.